CastleIO-Web/src/app/services/lobby.service.ts

```diff
@@ -0,0 +1,43 @@
+import {Injectable, OnInit} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {environment} from '../../environments/environment';
+import {Observable} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LobbyService {
+    baseUrl = environment.backendUrl + 'lobby/';
+    lobbyUrl: string;
+    lobbyEventSource: EventSource;
+
+    constructor(private http: HttpClient) {
+    }
+
+    // returns playerId
+    joinLobby(playerName: string): Observable<string> {
+        return this.http.put<string>(this.lobbyUrl, {}, {
+            params: {
+                playerName
+            }
+        });
+    }
```

Comment:
This request fails with 403 (Forbidden) because of CORS error:
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/lobby/a85372d8-87b8-48f4-9cf4-434c27161e9d/?playerName=eike. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).`

This is kind of weird because your CORS config is the same as in the ServerController but maybe a global CORS config file is needed as described here: https://spring.io/guides/gs/rest-service-cors/#global-cors-configuration

Reply:
fixed by https://github.com/soerenreichardt/CastlesIO/pull/13/commits/924c8ebc331a13f88a83c4a010008b1dde516666
the backend urls in the lobby service where missing the single requests url extension